contrib/copy-tex/index.html

```diff
@@ -1,17 +1,15 @@
 <!DOCTYPE html>
-<!--To run this example from a clone of the repository, run `make serve`
+<!--To run this example from a clone of the repository, run `npm start`
     in the root KaTeX directory and then visit with your web browser:
-    http://0.0.0.0:7936/contrib/copy-tex/index.html
+    http://localhost:7936/contrib/copy-tex/index.html
   -->
 <html>
   <head>
     <meta charset="UTF-8">
     <title>Copy-tex test</title>
     <script src="/katex.js" type="text/javascript"></script>
-    <link href="/katex.css" rel="stylesheet" type="text/css">
-    <link href="./copy-tex.css" rel="stylesheet" type="text/css">
```

Comment:
Why no .css files?

Reply:
I believe webpack will now inject these with either a script tag (extract plugin) or directly into the JS.